cpp/box/Box.h

```diff
@@ -435,6 +445,31 @@ class Box
         );
     }
 
+    //! Get mask of points that fit inside the box.
+    /*! \param points Point positions.
+        \param n_points The number of points.
+        \param cropped_mask Mask of points inside the box.
+    */
+    void crop(const vec3<float>* points, const unsigned int n_points,
+        unsigned char* cropped_mask) const
```

Comment:
Is there a reason we prefer `unsigned char*` over `bool*`?

Reply:
Extra support is required for boolean arrays. I wasn't sure if it was worth including an additional import line (`from libcpp cimport bool`) to create a mask .
I tried to use `bint`, but I couldn't get those results to appear correctly in numpy arrays so I settled on the smallest valid data type in both cython and C++ to represent 0/1 and converted it to `True`/`False` with python.